### DISCUSSION TITLE
Hi, does itwinsjs support geometric element movement now? At the frontend.

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#973] [Q&A] [answered]
Hi, does itwinsjs support geometric element movement now? At the frontend.

**Comment by pmconne** [comment, marked as answer]
Please consult [this sample](https://www.itwinjs.org/sample-showcase/?group=Viewer+Features&sample=explode-sample&imodel=House+Sample).
